docs/docs/configuration.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Configuration
+---
+
+Approzium is configured through environment variables.
+
+- `APPROZIUM_HOST`: Defaults to `"127.0.0.1"`.
+- `APPROZIUM_HTTP_PORT`: Defaults to `"6000"`.
+- `APPROZIUM_GRPC_PORT`: Defaults to `"6001"`.
+- `APPROZIUM_LOG_LEVEL`: Defaults to `"info"`. Supported selections are `"trace"`, `"debug"`,
+    `"info"`, `"warn"`, `"error"`, `"fatal"`, and `"panic"`. Upper case may be used.
+- `APPROZIUM_LOG_FORMAT`: Defaults to `"text"`. Also supports `"json"`.
+- `APPROZIUM_LOG_RAW`: Defaults to `"false"`, meaning that by default, Approzium's logs mask sensitive data.
+    Setting to `"true"` activates raw logs, so no sensitive values will be masked. For example, if set to `"true"`,
+    the AWS `signed_get_caller_identity` string will be fully included in logs, presenting an opportunity for
+    a viewer to impersonate another. Should only be set to `"true"` in environments where logs are carefully
+    guarded.
+- `APPROZIUM_VAULT_TOKEN_PATH`: Optional, if set it will cause the latest Vault token to always
+    be pulled from the given file.
+
+Approzium supports Vault for storing database credentials. To use Vault, at a minimum, the `VAULT_ADDR`
+must be set. Either the `VAULT_TOKEN` or `APPROZIUM_VAULT_TOKEN_PATH` must be set, with the `VAULT_TOKEN`
+taking precedence. We recommend using the `APPROZIUM_VAULT_TOKEN_PATH` with the Vault agent, as described
+[here](https://learn.hashicorp.com/vault/identity-access-management/vault-agent-aws), because that approach
+will allow the Vault token to be refreshed by the Vault agent, rather than eventually expiring.
+
+Additional Vault configuration is supported, as described
+[here](https://www.vaultproject.io/docs/commands#environment-variables).
```

Comment:
This configuration doc is super different from the current one due to a bunch of stuff being merged yesterday, needs a copy pasta of the current one.

Reply:
Thanks Becca! Yes, most of these were just placeholders I hadn't yet gotten around to changing.

> When I follow the Netlify preview link, I land on this page - would it be visible to others and live? Might need a tiny bit of tweaking.
> 
> <img alt="Screen Shot 2020-07-09 at 9 05 34 AM" width="1437" src="https://user-images.githubusercontent.com/1152361/87063630-78f00200-c1c3-11ea-879a-abce512b741e.png">
Definitely! This is all placeholder from docasaurus init -- working with sara on what we'll have and will have something solid by the end of the weekend.

> Also, the logo, I'm seeing the placeholder - could we purchase an image for use as a logo? Like some sort of clip art that refers to security? A lock, a key, a shield, a safe, something like that?
> 
> <img alt="Screen Shot 2020-07-09 at 9 07 52 AM" width="628" src="https://user-images.githubusercontent.com/1152361/87063937-e439d400-c1c3-11ea-91c2-5b8b9bcdc096.png">
Yes! Sara is buying the logo later today. 🙌

> Also, on the left nav, if you click on Client Libraries > Python, it takes you to https://www.psycopg.org/, but I was expecting it to take me to our python docs.  

this was a placeholder because I didn't have our readthedocs link on hand 😅, fixed now